CHANGELOG.md

```diff
@@ -1,15 +1,128 @@
 # Changelog
 
+## [0.57]
+
+Welcome to the 0.57 release of React Native! This release addresses a number of issues and has some exciting improvements. We've intentionally left this release in an extended release candidate state in order to improve quality. 
+
+You may notice that this release's changelog is thinner than previous versions; that's not from lack of hard work -- this release includes 566 commits by 78 different contributors! Instead, we've taken feedback and prepared a changelog that contains only user impacting changes. Please share your feedback and let us know how we can make this even more useful.
```

Comment:
Also, can you add the links to the `566 commits by 78 different contributors` as we did for last release (https://github.com/facebook/react-native/compare/0.55-stable...0.56-stable) and a link to the discussion 0.57.x Discussion for the feedback?